.github/workflows/cache-config-tests.yml

```diff
@@ -18,7 +18,8 @@
 name: Traffic Control Cache Config integration tests
 
 env:
-  RHEL_VERSION: 7
+  RHEL_VERSION: 8
+  ATS_VERSION: 8.1.x
```

Comment:
Setting the `ATS_VERSION` environment variable here does not do anything, because `docker-compose-ats-build.yml` does not configure the `trafficserver_build` service to inherit the `ATS_VERSION` environment variable, and the compose command in the *build-ats-test-rpm* GitHub Action does not pass it in:

https://github.com/apache/trafficcontrol/blob/a3f3e2045b86e734f52c043c59d9dfd73641b29e/.github/actions/build-ats-test-rpm/main.js#L23